policyengine_us/variables/gov/states/va/tax/income/deductions/va_itemized_deductions.py

```diff
@@ -7,8 +7,51 @@ class va_itemized_deductions(Variable):
     label = "Virginia itemized deductions"
     unit = USD
     definition_period = YEAR
-
-    reference = (
-        "https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/"
-    )
+    reference = "https://www.tax.virginia.gov/sites/default/files/taxforms/individual-income-tax/2021/schedule-2021.pdf"
     defined_for = StateCode.VA
+
+    def formula(tax_unit, period, parameters):
+        year = period.start.year
+        # Viginia applies the federal itemized deduction rate which was established before 2018
+        if year >= 2018 and year <= 2026:
+            instant_str = f"2017-01-01"
+
+        irs = parameters(instant_str).gov.irs.deductions.itemized.limitation
+        p = parameters(period).gov.irs.deductions.itemized.limitation
```

Comment:
@PavelMakarchuk Not very sure about this part since the `applicable_amount` is specific for 2021 and 2022 and no need to use pre-2018 values. Do I need to add pre-2018 values for `applicable_amount` as well?

Reply:
Yes I think that should apply to all of the IRS parameters, we did something similar in the Arkansas [PR](https://github.com/PolicyEngine/policyengine-us/pull/2822) where in the medical rate, where we need the 2013 values and it changed on the federal level by AR decided to continue the limit